src/CodeGen_LLVM.cpp

```diff
@@ -4935,5 +5134,108 @@ llvm::Constant *CodeGen_LLVM::get_splat(int lanes, llvm::Constant *value,
     return ConstantVector::getSplat(ec, value);
 }
 
+std::string CodeGen_LLVM::mangle_llvm_vector_type(llvm::Type *type) {
```

Comment:
Would be worth a short comment here explaining what the format looks like, with some examples

Reply:
Added detail in comment on the declaration in the header.